pyTigerGraph/pyTigerGraphAuth.py

```diff
@@ -231,7 +231,7 @@ def getToken(self, secret: str = None, setToken: bool = True, lifetime: int = No
         elif not(success) and not(secret):
             res = self._post(self.restppUrl+"/requesttoken", authMode="pwd", data=str({"graph": self.graphname}), resKey="results")
             success = True
-        else:
+        elif not(success) and (int(s) < 3 or (int(s) == 3 and int(m) < 5)):
```

Comment:
Seems like it is still missing the case when secret is given and the version is above 3.5. Do we use the secret to get token or use username/password?